space-cli/cmd/modules/accounts/commands.go

```diff
@@ -0,0 +1,80 @@
+package accounts
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/spaceuptech/space-cli/cmd/utils"
+)
+
+// Commands are the set of account commands for space-cli
+func Commands() []*cobra.Command {
+
+	var accountsCmd = &cobra.Command{
+		Use:   "accounts",
+		Short: "Operations for space-cloud accounts",
+	}
+
+	var viewAccountsCommand = &cobra.Command{
+		Use:   "view",
+		Short: "list all space-cloud accounts",
+		PreRun: func(cmd *cobra.Command, args []string) {
+			err := viper.BindPFlag("show-keys", cmd.Flags().Lookup("show-keys"))
+			if err != nil {
+				_ = utils.LogError("Unable to bind the flag ('show-keys')", nil)
+			}
```

Comment:
there will a silenceerr field which is a boolean set it to true, do the same for remaining 2 commands